### PR TITLE
Us interfaces

### PR DESCRIPTION
**Pull request by FlorianHe**
Adding interfaces and prepared datas to use while waiting for the back to be develop

### CODE REVIEW

**Review by Guillaum6e — DISMISSED** [review]

**Review by Guillaum6e — APPROVED** [review]

File: src/app/shared/datas/article.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Article } from "../interfaces/article";
+
+export const ARTICLE: Article = {
+    "id": 1,
+    "title": "Le caramel qui colle aux dents : pour ou contre ?",
```
Comment (Benja-san) [inline src/app/shared/datas/article.ts:5]: add a slug entry for better url navigation with slug

File: src/app/shared/datas/articles.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
+export const ARTICLES = [{
+    "id": 1,
+    "title": "Le caramel qui colle aux dents : pour ou contre ?",
```
Comment (Benja-san) [inline src/app/shared/datas/articles.ts:3]: use slug entry for better navigation. eg: slug: "le-caramel-qui-colle-aux-dents-pour-ou-contre"

File: src/app/shared/datas/categories.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export const CATEGORIES = [
+    {
+        "id" : 1,
+        "name" : "Fun"
```
Comment (Benja-san) [inline src/app/shared/datas/categories.ts:4]: use slug entry as well for url navigation

File: src/app/shared/interfaces/article.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { User } from './user'
+import { Category } from './category'
+
+export interface Article {
+
+    id: number
+    title: string
```
Comment (Benja-san) [inline src/app/shared/interfaces/article.ts:7]: add slug property

File: src/app/shared/interfaces/category.ts [hunk 5]
```diff
@@ -0,0 +1,4 @@
+export interface Category {
+    id : number
+    name : string
```
Comment (Benja-san) [inline src/app/shared/interfaces/category.ts:3]: add slug property

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by Guillaum6e — APPROVED** [review]

**Pull request merged**